docs/_advanced/error_handling.md

```diff
@@ -5,7 +5,7 @@ order: 1
 ---
 
 <div class="section-content">
-If an error occurs in a listener, it’s recommended you handle it directly. However, there are cases where errors may occur after your listener middleware has returned (such as `say()`, `respond()`, or not calling `ack()`). By default, these errors will be logged to the console. To handle them yourself, you can attach a global error handler to your app using the `error(fn)` method.
+If an error occurs in a listener, it’s recommended you handle it directly. However, there are cases where errors may be triggered after your listener middleware has already returned (such as in `say()`, `respond()`, or if you don't call `ack()` when it's required). By default, these errors will be logged to the console. To handle them yourself, you can attach a global error handler to your app with the `error(fn)` method.
```

Comment:
I think its important to be consistent with "listener" versus "listener middleware", and it seems like we are using both in the two clauses of this sentence.